README.md

```diff
@@ -2,8 +2,12 @@
 
 Tests and implementations for common data structures.
 
-Base repository for the [Core Data Structures](https://github.com/GuildCrafts/web-development-js/issues/127) goal.
+Base repository for the [Core Data Structures](https://github.com/GuildCrafts/web-development-js/issues/128) goal. See specs and quality rubric in the [CONTRACT.md](./CONTRACT.md) file.
 
 ## Installation and Setup
 
+_Fill this out_
+
 ## Usage and Examples
+
+_...and this_
```

Comment:
In order to prevent the pre-production commits from ending up in your review Prrr, when you follow that branch making procedure, instead of using the '|tail -1' console step, go into github and find the very first commit you made contributed by yourself, then use *that* SHA when creating the starting branch to pull master into. Find me if you want more info.